deploy-agent/tests/unit/deploy/common/test_stats.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Pinterest, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tests
+from time import sleep
+from deployd.common.stats import TimeElapsed
+
+
+class TestTimeElapsed(tests.TestCase):
+
+    def test__is_paused(self):
+        elapsed = TimeElapsed()
+        self.assertFalse(elapsed._is_paused())
+
+    def test_get(self):
+        elapsed = TimeElapsed()
+        then = elapsed.get()
+        self.assertFalse(elapsed._is_paused())
```

Comment:
i think this is covered in other tests? lmk if incorrect, it looks like a lot of the adjacent asserts can be removed to have each test focus on one aspect

Reply:
trying to make each test work in isolation - we can revisit in a future diff if you feel strongly about a different pattern

Reply:
nonblocking